l10n_it_fatturapa_out_welfare/views/welfare_fund_type_amount_views.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  ~ Copyright 2022 Simone Rubino - TAKOBI
+  ~ License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+  -->
+
+<odoo>
+    <!-- Views -->
+    <record id="welfare_fund_type_amount_view_form" model="ir.ui.view">
+        <field name="name">Form View for Welfare Fund Type Amount</field>
+        <field name="model">welfare.fund.type.amount</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="name"/>
+                        <field name="welfare_fund_type_id"/>
+                        <field name="amount"/>
+                        <field name="administration_reference"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="welfare_fund_type_amount_view_tree" model="ir.ui.view">
+        <field name="name">Tree View for Welfare Fund Type Amount</field>
+        <field name="model">welfare.fund.type.amount</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name"/>
+                <field name="welfare_fund_type_id"/>
+                <field name="amount"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="welfare_fund_type_amount_view_search" model="ir.ui.view">
+        <field name="name">Search View for Welfare Fund Type Amount</field>
+        <field name="model">welfare.fund.type.amount</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="welfare_fund_type_id"/>
+                <field name="amount"/>
+            </search>
+        </field>
+    </record>
+
+    <!-- Actions -->
+    <record id="welfare_fund_type_amount_action" model="ir.actions.act_window">
+        <field name="name">Welfare Fund Type Amounts</field>
+        <field name="res_model">welfare.fund.type.amount</field>
+    </record>
+
+    <!-- Menus -->
+    <record id="welfare_fund_type_amount_menu" model="ir.ui.menu">
+        <field name="name">Welfare Fund Type Amounts</field>
+        <field name="action" ref="welfare_fund_type_amount_action"/>
+        <field name="parent_id" ref="l10n_it_fatturapa.menu_fattura_pa_receivables"/>
```

Comment:
Questo andrebbe messo in 
Fatturazione > configurazione > contabilità

Reply:
Ho trovato il menù
> Fatturazione > Configurazione > Contabilità > Fiscalità italiana > Fatturazione elettronica

dedicato alle configurazioni della FE e l'ho spostato lì sotto, puoi controllare se ora va meglio?